databases/de-database-schema/src/main/conversions/v1.4.0/c140_2012052501.clj

```diff
@@ -1,134 +0,0 @@
-         LEFT JOIN transformation_steps ts
-             ON tts.transformation_step_id = ts.id
-         LEFT JOIN transformations tx ON ts.transformation_id = tx.id
-         LEFT JOIN template t ON tx.template_id = t.id
-         LEFT JOIN deployed_components dc ON t.component_id = dc.id
-    GROUP BY analysis.hid,
-             analysis.id,
-             analysis.name,
-             analysis.description,
-             integration.integrator_name,
-             integration.integrator_email,
-             analysis.integration_date,
-             analysis.edited_date,
-             analysis.wikiurl,
-             analysis.deleted,
-             analysis.disabled"))
-
-(defn- create-analysis-job-types
-  "Creates the analysis_job_types view, which lists all of the job types
-   associated with an analysis."
-  []
-  (println "\t* defining analysis_job_types view")
-  (exec-raw
-   "CREATE VIEW analysis_job_types AS
-    SELECT a.hid AS analysis_id,
-           dc.type AS job_type
-    FROM transformation_activity a
-        JOIN transformation_task_steps tts
-             ON a.hid = tts.transformation_task_id
-        JOIN transformation_steps ts ON tts.transformation_step_id = ts.id
-        JOIN transformations tx ON ts.transformation_id = tx.id
-        JOIN template t ON tx.template_id::text = t.id::text
-        JOIN deployed_components dc ON t.component_id::text = dc.id::text"))
-
-(defn- redefine-version-table
-  "Redefines the version table."
-  []
-  (println "\t* redefining version table")
-  (exec-raw "DROP TABLE version")
-  (exec-raw "CREATE SEQUENCE version_id_seq")
-  (exec-raw
-   "CREATE TABLE version (
-        id bigint DEFAULT nextval('version_id_seq'),
-        version character varying(20) NOT NULL,
-        applied timestamp DEFAULT now(),
-        PRIMARY KEY (id)
-    )"))
-
-(defn- drop-obsolete-tables
-  "Drops obsolete tables from the database."
-  []
-  (println "\t* dropping obsolete tables")
-  (exec-raw "DROP TABLE block_type")
-  (exec-raw "DROP TABLE file_type")
-  (exec-raw "ALTER TABLE workspace DROP CONSTRAINT workspace_home_folder_fkey")
-  (exec-raw "DROP TABLE folder_folder")
-  (exec-raw "DROP TABLE folder")
-  (exec-raw "DROP TABLE IF EXISTS precedence_constraints")
-  (exec-raw "DROP SEQUENCE precedence_constraints_id_seq")
-  (exec-raw "DROP TABLE predicate_lookup")
-  (exec-raw "DROP TABLE provenance_device")
-  (exec-raw "DROP SEQUENCE seq_classificationid_classification")
-  (exec-raw "DROP SEQUENCE seq_name_nameid")
-  (exec-raw "DROP SEQUENCE seq_namesource_nameid")
-  (exec-raw "DROP TABLE statement_type")
-  (exec-raw "DROP SEQUENCE step_precedence_id_seq")
-  (exec-raw "DROP TABLE thing_type_code"))
-
-(defn convert
-  "Performs the conversion for database version 1.4.0:20120525.01."
-  []
-  (println "Performing conversion for" version)
-  (add-edited-date)
-  (redefine-analysis-listing)
-  (create-analysis-job-types)
-  (redefine-version-table)
-  (drop-obsolete-tables))
```

Comment:
The chances of these old conversions ever being run again are very remote, so I removed them.